docs/book/features/vsphere_csi_migration.md

```diff
@@ -0,0 +1,175 @@
+- Following vSphere in-tree StorageClass parameters will not be supported after enabling migration.
+  - `hostfailurestotolerate`
+  - `forceprovisioning`
+  - `cachereservation`
+  - `diskstripes`
+  - `objectspacereservation`
+  - `iopslimit`
+  - `diskformat`
+- Storage Policy should not be renamed or deleted after enabling migration. Volume migration requires storage policy to be present to successfully register volume as Container Volume in vSphere.
+- Datastore should not be renamed after enabling migration. Volume migration relies on the datastore name retrieved from the volume path available in the Persistent Volume Source.
+- For statically created vSphere in-tree Persistent Volume Claims and Persistent Volumes, make sure to add following annotations. Statically provisioned in-tree vSphere volumes can't be migrated to CSI without following these annotations.
+
+    Annotation on PV
+
+      annotations:
+        pv.kubernetes.io/provisioned-by: kubernetes.io/vsphere-volume
+
+    Annotation on PVC
+
+      annotations:
+        volume.beta.kubernetes.io/storage-provisioner: kubernetes.io/vsphere-volume
+
+- vSphere CSI Driver does not support Provisioning `eagerzeroedthick` and `zeroedthick` volume. After the migration is enabled, when a new volume is requested using the in-tree provisioner and `diskformat` parameter, `diskformat` parameter is dropped by vSphere CSI Driver.
+- vSphere CSI Driver does not support raw vSAN policy parameters. After the migration is enabled, when a new volume is requested using in-tree provisioner and vSAN raw policy parameters specified in the StorageClass, Volume Creation will be failed by vSphere CSI Driver.
+- vSphere CSI Migration requires vSphere 7.0u1. Customers who have in-tree vSphere volumes must upgrade vSphere to 7.0u1. Customers who do not need to migrate in-tree vSphere volumes can use vSphere 67u3 and above.
+- vSphere CSI driver does not support volumes formatted with the Windows file system. Migrated in-tree vSphere volumes using the windows file system can't be used with vSphere CSI driver.
+- In-tree vSphere volume plugin is heavily relying on the name of the datastore set on the PV’s Source. After migration is enabled, Storage DRS should not be enabled. If Storge DRS moves disk from one datastore to another further volume operations may break.
+
+## How to enable vSphere CSI Migration
+
+In Kubernetes 1.19 release, vSphere CSI Migration is available with `beta` feature-gates.
+
+To try out vSphere CSI migration in beta for vSphere plugin, perform the following steps.
+
+1. Upgrade vSphere to 7.0u1.
+2. Upgrade kubernetes to 1.19 release.
+3. Install vSphere CSI Driver. Use deployment manifests and RBAC files published at https://github.com/kubernetes-sigs/vsphere-csi-driver/tree/master/manifests/latest/vsphere-7.0u1/vanilla
```

Comment:
Install CSI driver >2.1.0.
Latest version can be deployed using the deployment manifests and RBAC files published at https://github.com/kubernetes-sigs/vsphere-csi-driver/tree/master/manifests/latest/vsphere-7.0u1/vanilla/.

Reply:
changed URL to - https://github.com/kubernetes-sigs/vsphere-csi-driver/tree/master/manifests/v2.1.0/vsphere-7.0u1/vanilla